base/intfuncs.jl

```diff
@@ -328,7 +331,11 @@ const HWNumber = Union{HWReal, Complex{<:HWReal}, Rational{<:HWReal}}
 # be computed in a type-stable way even for e.g. integers.
 @inline function literal_pow(f::typeof(^), x, ::Val{p}) where {p}
     if p < 0
-        literal_pow(^, inv(x), Val(-p))
+        if x isa BitInteger64
+            f(Float64(x), p)
```

Comment:
Maybe add some comment on this?  Is this to improve performance, accuracy, or both?

Reply:
Good idea. The reasoning is that computing `inv(x)^(-p)` rounds twice. Once to compute `inv(x)` and once to compute the power. Since the `Float64^Float64` code works to half an ULP (even for negative numbers), we want to let that code handle the negative powers since it does so better.